XamlControlsGallery/ControlPages/PipsPagerPage.xaml

```diff
@@ -0,0 +1,85 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.PipsPagerPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AppUIBasics"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel>
+        <local:ControlExample x:Name="Example1" HeaderText="PipsPager with options to change its orientation and button visibility.">
```

Comment:
I'd recommend moving this sample to the end. Our general approach is to have the more "real world" examples at the top, and the "API in action" samples at the bottom (as seen in NavView).